batch_processing.sh

```diff
@@ -93,6 +93,8 @@ sct_process_segmentation -i t2_seg.nii.gz -vert 2:3 -o csa_c2c3.csv
 sct_detect_pmj -i t2.nii.gz -c t2 -qc "$SCT_BP_QC_FOLDER" 
 # Compute cross-section area at 60 mm from PMJ averaged on a 30 mm extent
 sct_process_segmentation -i t2_seg.nii.gz -pmj t2_pmj.nii.gz -pmj-distance 60 -pmj-extent 30 -qc "$SCT_BP_QC_FOLDER" -qc-image t2.nii.gz -o csa_pmj.csv
+# Compute morphometrics in PAM50 anatomical dimensions
+sct_process_segmentation -i t2_seg.nii.gz -vertfile t2_seg_labeled.nii.gz -perslice 1 -normalize-PAM50 1 -o csa_pam50.csv
```

Comment:
I notice that in this sample command, we use `-vertfile t2_seg_labeled.nii.gz`.

But, for `sct_process_segmentation`, the default value for `-vertfile` is `./label/template/PAM50_levels.nii.gz`. (i.e. the template level file warped to the subject space).

The two labeled segmentations are very similar, but not equal:

![ezgif-5-00081ef359](https://user-images.githubusercontent.com/16181459/215529759-9933ccf0-6ad1-49fd-9e75-f466f456314c.gif)

What is the difference between using the label files generated by `sct_label_vertebrae` vs the label files generated by `sct_warp_template`? Is there a "correct" choice for performing the normalization here?

Reply:
no necessarly, in this case, the registration to template was really good, but not all analysis require registration to template, so just using  `sct_label_vertebrae` is fine for this!